ieeg_visual_multimodal/sub-som682/ses-nyu3t01/func/sub-som682_ses-nyu3t01_task-hrf_run-1_events.tsv

```diff
@@ -0,0 +1,33 @@
+onset	duration	trial_type	trial_name	stim_file	stim_file_index
+29.75	0.2	1	HRFPATTERN	sub-som682_ses-nyu3t01_task-hrfpattern_run-1.mat	8
```

Comment:
OK, we don't actually use .png files in our stimulus presentation, the stimuli are drawn by PsychToolbox directly from the .mat file, with separate .mat files for each task. But I can write out 'fake' png files for all the different stimuli from the different tasks and update the events.tsv (that will generate a lot of png files though because each task has different stimuli).

Reply:
I replaced the stimuli with empty .png files and updated the stim_file columns in the events.tsv files accordingly